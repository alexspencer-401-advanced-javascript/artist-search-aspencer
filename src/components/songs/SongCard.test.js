import React from 'react';
import { shallow } from 'enzyme';
import SongCard from './SongCard';

describe('SongCard component', () => {
  it('renders SongCard', () => {
    const wrapper = shallow(<SongCard id={'test string'} title={'test string'} name={'test string'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
