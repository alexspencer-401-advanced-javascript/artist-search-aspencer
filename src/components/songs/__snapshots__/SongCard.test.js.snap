// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SongCard component renders SongCard 1`] = `
<div
  key="test string"
>
  <Link
    to="/lyrics/test string/test string"
  >
    <p>
      test string
    </p>
  </Link>
</div>
`;
